src/Catalyst.Core.Lib.Tests/IntegrationTests/Network/DnsIntergrationTests.cs

```diff
@@ -0,0 +1,50 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Catalyst.TestUtils;
+using DnsClient;
+using DnsClient.Protocol;
+using FluentAssertions;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.IntegrationTests.Network
+{
+    public sealed class DnsIntegrationTests
+    {
+        [Fact]
+        [Trait(Traits.TestType, Traits.IntegrationTest)]
+        public async Task GetTxtRecords_should_return_seeds()
+        {
+            var trueClient = new LookupClient(new IPEndPoint(IPAddress.Parse("9.9.9.9"), 53));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make sure using this hardcoded IP address '9.9.9.9' is safe here.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)